Home Page Empty Database Fix

leong96 opened the pull request:
This commit uses try-catch to go past the ValueError, which is thrown if the database is empty.

leong96 committed:
Update views.py

This commit uses try-catch to go past the ValueError, which is thrown if the database is empty.

nlMeminger commented on grasa_event_locator/views.py:
```diff
@@ -263,52 +263,55 @@ def index(request):
+                #        lat2, lng2 = value.split(', ')
+                #        lat.append(lat2)
+                #        lng.append(lng2)
+                try:
```
try's should not be nested if avoidable.  Ideally you would only have the code that can cause the exception in the try block, and everything else outside.

leong96 replied:
Got it. Removed the inner try block, and added the exception it was catching to the remaining try block.

leong96 committed:
Try block

Removed the inner try so there is no longer any nested try's. The block now catches the two known exceptions.

leong96 committed:
Merge branch 'master' into hleong-back-end

R-Eli-Levasseur reviewed (approved):
Looks good
